test/Command.js

```diff
@@ -187,7 +184,70 @@ describe("Command", () => {
   });
 
   describe(".runPreparations()", () => {
-    it("needs tests");
+    describe(".filteredPackages", () => {
+      let testDir;
+
+      beforeEach(() => initFixture("UpdatedCommand/basic").then((dir) => {
+        testDir = dir;
+      }));
+
+      class TestCommand extends Command {}
+
+      function cli(cmd, ...args) {
```

Comment:
I saw similar helpers in the tests for the `updated` command, however, in order to reduce the amount of changes, I decided to write a few extra lines of code rather than refactoring them out into a common lib.